### ISSUE TITLE
Continue without connection to server for output.

### ISSUE THREAD

**Issue by JamesBremner** [JamesBremner/in2out#2]
If data is received when there is no server connection, it is discarded.

**Comment by JamesBremner** [comment]
I want the app try to connect to the receiver every time the app receive new data.

**Comment by JamesBremner** [comment]
Starting the app when the output receiver is not running

```
>in2out --input 27678 --output 127.0.0.1:27654
in2out built Aug 12 2022 15:24:05
Waiting for input on port 27678
looking for output server 127.0.0.1:27654
listening for client on port 27678
No connection could be made because the target machine actively refused it.  Generally, it happens that something is preventing a connection to the port or hostname.  Either there is a firewall blocking the connection  or the process that is hosting the service is not listening on that specific port.
wex::tcp failed connection to server
client 127.0.0.1 accepted
connected
Input connected on 27678
```

so at this point, the app retries to connect to the missing server whenever it receives some input

```
Input: Hello
Input Hex: 48 65 6c 6c 6f

Output Hello modified
no-one is listening
looking for output server 127.0.0.1:6c06
No connection could be made because the target machine actively refused it.  Generally, it happens that something is preventing a connection to the port or hostname.  Either there is a firewall blocking the connection  or the process that is hosting the service is not listening on that specific port.
wex::tcp failed connection to server
```

Switch on the server and send some more data

```
Input: Hello
Input Hex: 48 65 6c 6c 6f
Output Hello modified
no-one is listening
looking for output server 127.0.0.1:6c06
No connection could be made because the target machine actively refused it.  Generally, it happens that something is preventing a connection to the port or hostname.  Either there is a firewall blocking the connection  or the process that is hosting the service is not listening on that specific port.
wex::tcp failed connection to server
Input: Hello
Input Hex: 48 65 6c 6c 6f

Output Hello modified
no-one is listening
looking for output server 127.0.0.1:6c06
wex::tcp connected to server
```

When the app tried once more to connect, it found and connected succesfully.

Now, when data arrives it is transmitted successfully

```
Input: Hello
Input Hex: 48 65 6c 6c 6f

Output Hello modified
wex::tcp::send Hello modified
```

**Comment by JamesBremner** [comment]
v1.1 released.  You can download it from

https://github.com/JamesBremner/in2out/releases/tag/v1.1

**Comment by RmanSC** [comment]
Issue can be closed, as assosiated issues #7 and #9 are fixed
